### DISCUSSION TITLE
Feature Request - JSON Encoded options in the database

### DISCUSSION THREAD

**Discussion by JMar323** [snicco/fortress#9] [Feature Requests]
In addition to vnp working with a serialized string, add functionality to encrypt options that are json encoded in the database.

**Comment by calvinalkan** [comment]
Good idea, 

Something like this might work; I don't think we can decide what's JSON and what's not at runtime.

````json
{
  "vaults_and_pillars": {
    "some_setting": {
      "type": "json"
    }
  }
}
````

**Comment by calvinalkan** [comment]
In the meantime, you can make the entire option a vault. 

That will work without issues, but if you have HUGE scale, it might be a performance concern.
